Add prerequisites section to configure the GitLab App (#444)
pulumi/pulumi-hugo: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md

* [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
* If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)

## Prerequisites

* You must have a GitLab identity associated with your Pulumi account.

> * If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}}).

Author: ```suggestion
* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}})
```

Author: Looks like the line-break deceived us into thinking this was the end of the line

* [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
* If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)

## Prerequisites

> * You must have a GitLab identity associated with your Pulumi account

Reviewer: ```suggestion
* You must have a GitLab identity associated with your Pulumi account.
```

> * If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}})

Reviewer: ```suggestion
* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}}).
```

  in Pulumi.

>   * After you add the organization ensure that it uses GitLab as its identity provider. You may [change your org identity]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}})
>     provider in the Pulumi Console after the organization has been added.

Reviewer: ```suggestion
  * After you add the organization, ensure that it uses GitLab as its identity provider. You may [change your org identity provider]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}}) in the Pulumi Console under Settings > Access Management > Membership Requirements.
```

Reviewer 2: seems a bit weird to link to the content that describes that as well as describing it here? i think we should prolly choose one or the other.

Reviewer: Ah, fair point -- maybe something like this, then:

```suggestion
  * After you add the organization [ensure that it uses GitLab as its identity provider]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}}). 
```
